ceasiompy/ActuatorDisk/tests/test_actuatordisk.py

```diff
@@ -0,0 +1,157 @@
+"""
+CEASIOMpy: Conceptual Aircraft Design Software
+
+Developed by CFS ENGINEERING, 1015 Lausanne, Switzerland
+
+Test functions for 'lib/ActuatorDisk/actuatordisk.py'
+
+Python version: >=3.8
+
+| Author : Giacomo Benedetti
+| Creation: 2022-11-03
+
+"""
+
+# =================================================================================================
+#   IMPORTS
+# =================================================================================================
+
+
+from pathlib import Path
+
+from ceasiompy.ActuatorDisk.func.optimalprop import thrust_calculator
+from ceasiompy.ActuatorDisk.func.optimalprop import axial_interference_function
+import numpy as np
+from ceasiompy.utils.ceasiompyutils import get_results_directory
+
+
+from pytest import approx
+
+MODULE_DIR = Path(__file__).parent
+CPACS_IN_PATH = Path(MODULE_DIR, "ToolInput", "simpletest_cpacs.xml")
+CPACS_OUT_PATH = Path(MODULE_DIR, "ToolOutput", "ToolOutput.xml")
+ACTUATOR_DISK_PATH = Path(MODULE_DIR, "ToolOutput")
+
+# =================================================================================================
+#   CLASSES
+# =================================================================================================
+
+
+# =================================================================================================
+#   FUNCTIONS
+# =================================================================================================
+
+
+def test_axial_interference():
+
+    lagrangian_molt = np.array([0.3, 0.12, 0.05])
+    adimensional_radius = np.array([0.25, 0.5, 0.75])
+
+    calc_axial_interference = np.array([0.01699, 0.01994, 0.01689])
+
+    axial_interference_factor = axial_interference_function(lagrangian_molt, adimensional_radius)
+
+    assert all(axial_interference_factor) == all(calc_axial_interference)
+
+
+def test_check_output():
+
+    stations = np.array([20, 20, 20, 20, 20, 20, 20, 20])
+    total_thrust_coefficient = np.array([0.5, 0.8, 1, 1.2, 1.5, 0.2, 0.15])
+    radius = np.array([1.5, 1.5, 1.2, 1.4, 2, 1.4, 1.4, 1.7])
+    hub_radius = np.array([0.2, 0.15, 0.1, 0.1, 0.2, 0.1, 0.1, 0.2])
+    advanced_ratio = np.array([1.5, 2, 1.8, 1.6, 1.8, 1.4, 1.3, 1.7])
+    free_stream_velocity = np.array([150, 190, 180, 140, 190, 130, 130, 160])
+    prandtl = np.array(
```

Comment:
I think you could try to use a dictionary to store these values and iterate in it.